app.js

```diff
@@ -4,13 +4,28 @@ const path = require('path');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
 const CORS = require('./src/middlewares/cors');
+const cron = require('node-cron');
+const controller = require('./src/components/upload/controller');
 require('dotenv').config();
 
 const news_api = require('./src/components/news-api/index');
 const indian_stats_api = require('./src/components/indian-stats-api/index');
 
 const app = express();
 
+require('./db/connection');
+
+// fetch data every 15 minutes
+cron.schedule("0 */15 * * * *",(req,res,next)=>{
```

Comment:
Let's put this code in index file of this component because you are already referencing the index file here there is no point making the app.js file too cluttered.

Reply:
additionally you will not get `req`,`res` and `next` in that function, it's **wrong implementation**

@Dev-Aman7 maybe you can help @ayush5588 understand how `cron` works.